doc/api/util.md

```diff
@@ -1202,7 +1724,8 @@ added: v0.11.5
 deprecated: v4.0.0
 -->
 
-> Stability: 0 - Deprecated
+> Stability: 0 - Deprecated:
+> Use `typeof value !== 'object' && typeof value !== 'function'` instead.
```

Comment:
The function is implemented as `value === null || typeof value !== 'object' && typeof value !== 'function'`.